lib/pundit/railtie.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Pundit
+  class Railtie < Rails::Railtie
+    if Rails.version.to_f >= 8.0
+      initializer "pundit.stats_directories" do
+        require "rails/code_statistics"
+
+        Rails::CodeStatistics.register_directory(
+          "Policies", Rails.root.join("app/policies").to_s
+        )
+
+        %w[test/policies spec/policies].each do |dir|
+          path = Rails.root.join(dir)
+
+          Rails::CodeStatistics.register_directory("Policies tests", path.to_s, test_directory: true) if path.exist?
```

Comment:
A _tiny_ nitpick, it seems the documentation you linked to has _singular_ for this, e.g. `Models` and `Model tests`.

Reply:
done!